src/Uno.UWP/Globalization/ApplicationLanguages.cs

```diff
@@ -22,9 +22,9 @@ public static string PrimaryLanguageOverride
 			get
 			{
 				if (_primaryLanguageOverride.Length == 0 &&
-					ApplicationData.Current.LocalSettings.Values[PrimaryLanguageOverrideSettingKey] is string savedValue)
+					ApplicationData.Current.LocalSettings.Values.TryGetValue(PrimaryLanguageOverrideSettingKey, out var savedValue))
 				{
-					_primaryLanguageOverride = savedValue;
+					_primaryLanguageOverride = (string)savedValue;
```

Comment:
We should still be validating is the setting value is actually a string, in case it gets corrupted.

Reply:
```suggestion
					_primaryLanguageOverride = stringSavedValue;
```